src/pages/iou/request/IOURequestStartPage.tsx

```diff
@@ -23,46 +22,31 @@ import type {IOURequestType} from '@userActions/IOU';
 import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
 import type SCREENS from '@src/SCREENS';
-import type {Policy, Report, SelectedTabRequest, Transaction} from '@src/types/onyx';
 import {isEmptyObject} from '@src/types/utils/EmptyObject';
 import IOURequestStepAmount from './step/IOURequestStepAmount';
 import IOURequestStepDistance from './step/IOURequestStepDistance';
 import IOURequestStepScan from './step/IOURequestStepScan';
 import type {WithWritableReportOrNotFoundProps} from './step/withWritableReportOrNotFound';
 
-type IOURequestStartPageOnyxProps = {
-    /** The report that holds the transaction */
-    report: OnyxEntry<Report>;
-
-    /** The policy tied to the report */
-    policy: OnyxEntry<Policy>;
-
-    /** The tab to select by default (whatever the user visited last) */
-    selectedTab: OnyxEntry<SelectedTabRequest>;
-
-    /** The transaction being modified */
-    transaction: OnyxEntry<Transaction>;
-
-    /** The list of all policies */
-    allPolicies: OnyxCollection<Policy>;
-};
-
-type IOURequestStartPageProps = IOURequestStartPageOnyxProps & WithWritableReportOrNotFoundProps<typeof SCREENS.MONEY_REQUEST.CREATE>;
+type IOURequestStartPageProps = WithWritableReportOrNotFoundProps<typeof SCREENS.MONEY_REQUEST.CREATE>;
 
 function IOURequestStartPage({
-    report,
-    policy,
     route,
     route: {
         params: {iouType, reportID},
     },
-    selectedTab,
-    transaction,
-    allPolicies,
 }: IOURequestStartPageProps) {
     const styles = useThemeStyles();
     const {translate} = useLocalize();
     const [isDraggingOver, setIsDraggingOver] = useState(false);
+    const [report] = useOnyx(`${ONYXKEYS.COLLECTION.REPORT}${reportID}`);
+    // eslint-disable-next-line  @typescript-eslint/prefer-nullish-coalescing
+    const [policy] = useOnyx(`${ONYXKEYS.COLLECTION.POLICY}${report?.policyID || -1}`);
+    const [selectedTab] = useOnyx(`${ONYXKEYS.COLLECTION.SELECTED_TAB}${CONST.TAB.IOU_REQUEST_TYPE}`);
+    // eslint-disable-next-line  @typescript-eslint/prefer-nullish-coalescing
+    const [transaction] = useOnyx(`${ONYXKEYS.COLLECTION.TRANSACTION_DRAFT}${route?.params.transactionID || -1}`);
+    const [allPolicies] = useOnyx(ONYXKEYS.COLLECTION.POLICY);
```

Comment:
This seems like kind of a lot of data to subscribe to, but you didn't add it so NAB.